docs/topics/wildfly-swarm/docs/reference/fractions/management.adoc

```diff
@@ -41,14 +41,17 @@ The path of the audit log file.
 swarm.management.audit-access.file-handlers._KEY_.relative-to:: 
 The name of another previously named path, or of one of the standard paths provided by the system. If 'relative-to' is provided, the value of the 'path' attribute is treated as relative to the path specified by this attribute.
 
+swarm.management.audit-access.file-handlers._KEY_.rotate-at-startup:: 
+Whether the old log file should be rotated at server startup.
+
 swarm.management.audit-access.in-memory-handlers._KEY_.max-history:: 
 The maximum number of operation stored in history for this handler.
 
 swarm.management.audit-access.json-formatters._KEY_.compact:: 
 If true will format the JSON on one line. There may still be values containing new lines, so if having the whole record on one line is important, set escape-new-line or escape-control-characters to true.
 
 swarm.management.audit-access.json-formatters._KEY_.date-format:: 
-The date format to use as understood by {@link java.text.SimpleDateFormat}. Will be ignored if include-date="false".
+The date format to use as understood by java.text.SimpleDateFormat. Will be ignored if include-date="false".
```

Comment:
full classpath should use backticks

Reply:
Note to self: not easily fixable in our sources.